Sources/String+Sweetness.swift

```diff
@@ -1,5 +1,14 @@
+import Foundation
+
 public extension String {
     public var length: Int {
         return self.characters.count
     }
+
+    public var isValidEmail: Bool {
+        let emailRegEx = "[A-Z0-9a-z._%+-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,}"
```

Comment:
StackOverflow is sadly a terrible repository for this sort of stuff. 😢 It has a hell of a bias for quick dirty solutions.